BlazarBase/src/main/java/com/hubspot/blazar/base/feedback/Feedback.java

```diff
@@ -0,0 +1,48 @@
+package com.hubspot.blazar.base.feedback;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Optional;
+
+public class Feedback {
+
+  private final String username;
+  private final String message;
+  private final String page;
+  private final long timestamp;
```

Comment:
I think we can drop `timestamp` from here and just do `System.currentTimeMillis()` in the resource